rust/integration-tests/src/sdk_tests/mod.rs

```diff
@@ -7,15 +7,19 @@ use aptos_indexer_testing_framework::{
 use assert_json_diff::assert_json_eq;
 use diesel::{Connection, PgConnection};
 use serde_json::Value;
-use std::{collections::HashMap, fs, path::Path};
+use std::{
+    collections::HashMap,
+    fs,
+    path::{Path, PathBuf},
+};
 
 #[cfg(test)]
 pub mod events_processor_tests;
 #[cfg(test)]
 pub mod fungible_asset_processor_tests;
 
 #[allow(dead_code)]
-pub const DEFAULT_OUTPUT_FOLDER: &str = "sdk_expected_db_output_files/";
+pub const DEFAULT_OUTPUT_FOLDER: &str = "sdk_expected_db_output_files2";
```

Comment:
Did you mean to change this?

Reply:
I wanted to show what the new folder structure would look like so that we can compare. However, since we are going to keep the original version as a folder name, you can ignore this!